azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -117,6 +137,29 @@ def set_one_param(self, name: str, value: Any):
             params = self.params["params"] if self.nested_params else self.params
             params[name] = str(value) if self.force_str_params else value
 
+    def check_set_schedule(self, schedule:RangeSchedule, schedule_name:str):
```

Comment:
please add a whitespace after semicolon in accordance with PEP8 guidelines: https://www.python.org/dev/peps/pep-0008/#variable-annotations
This method is missing a docstring.

Reply:
docstring are added.